pkg/connector/view.go

```diff
@@ -66,6 +71,12 @@ func (r *viewSyncer) List(ctx context.Context, parentResourceID *v2.ResourceId,
 }
 
 func (r *viewSyncer) Entitlements(ctx context.Context, resource *v2.Resource, pToken *pagination.Token) ([]*v2.Entitlement, string, annotations.Annotations, error) {
+	_, _, err := parseWithDatabaseID(resource.Id.Resource)
+	if err != nil {
+		return nil, "", nil, err
+	}
+
+	// TODO: add dat
```

Comment:
is this done?

Reply:
No, I need to add database name prefix for this entitlement, thanks